aiohttp/helpers.py

```diff
@@ -115,6 +116,25 @@ def strip_auth_from_url(url):
 
 
 def proxies_from_env():
+    netrc_obj = None
+    netrc_path = os.environ.get('netrc')
+    if netrc_path is None:
+        home_dir = os.path.expanduser('~')
```

Comment:
Could you use `pathlib` instead of `os.path`?
E.g. https://docs.python.org/3/library/pathlib.html#pathlib.Path.home for getting user home folder.